src/views/movies.js

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import { fetchMovies } from '../services/fetchmovies';
+
+export default function Movies() {
+  const [movies, setMovie] = useState([]);
+  const [search, setSearch] = useState('');
+  const [change, setChange] = useState('');
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchMovies(`${search}`);
+      setMovie(data);
+      // console.log('Movie data', data);
+    };
+    fetchData();
+  }, [search]);
+  return (
+    <div>
+      <div>Star Wars Movies</div>
+      <form
+        onSubmit={(e) => {
+          e.preventDefault();
+          // console.log('clicked');
+          setSearch(change);
```

Comment:
Nice job having a onSubmit function so that constant requests aren't made to supabase. Consider moving the form into a presentational component for organizing code based on separation of concerns.